### ISSUE TITLE
0.4 release on pypi is broken

### ISSUE THREAD

**Issue by dhavlik** [gocept/zodb.py3migrate#6]
Got this when installing the egg via buildout:

```
Installing py3migrate.
Getting distribution for 'zodb.py3migrate'.
error: [Errno 2] No such file or directory: '/var/folders/bq/1z48c2254r3dh3p24bpj_cq40000gn/T/easy_install-a1AYlQ/zodb.py3migrate-0.4/HACKING.rst'
An error occurred when trying to install zodb.py3migrate 0.4. Look above this message for any errors that were output by easy_install.
```

**Comment by icemac** [comment]
Adding a MANIFEST.in could help to prevent this in future.

**Comment by sallner** [comment]
solved by #8.
